src/Parser/ParserInterface.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace React\Http\Parser;
+
+use Evenement\EventEmitterInterface;
+use React\Http\Request;
+
+interface ParserInterface extends EventEmitterInterface
+{
+    public function __construct(Request $request);
```

Comment:
The intent does not appear to be particularly clear, so what exactly is this interface supposed to do? (input parameters, expected output etc.)